src/main/java/games/strategy/engine/data/SerializationProxySupport.java

```diff
@@ -3,43 +3,54 @@
 
 /**
  * Annotation to mark magic 'writeReplace' methods that are used to support the serialization proxy pattern:
- * - https://github.com/triplea-game/triplea/issues/814
- * - http://blog.codefx.org/design/patterns/serialization-proxy-pattern/
- * - https://dzone.com/articles/serialization-proxy-pattern
- * - http://vard-lokkur.blogspot.com/2014/06/serialization-proxy-pattern-example.html
- * - https://youtu.be/V1vQf4qyMXg?t=56m12s
  *
+ * <ul>
+ * <li>https://github.com/triplea-game/triplea/issues/814</li>
+ * <li>http://blog.codefx.org/design/patterns/serialization-proxy-pattern/</li>
+ * <li>https://dzone.com/articles/serialization-proxy-pattern</li>
+ * <li>http://vard-lokkur.blogspot.com/2014/06/serialization-proxy-pattern-example.html</li>
+ * <li>https://youtu.be/V1vQf4qyMXg?t=56m12s</li>
+ * </ul>
+ *
+ * <p>
  * A typical usage of this annotation will and pattern will look like this:
- * Code formatted below for easy copy paste:<code>
-     &#64;SerializationProxySupport
-     protected Object writeReplace() {
-       return new SerializationProxy(this);
-     }
-
-     private static class SerializationProxy implements Serializable {
-       private static final long serialVersionUID = -4193924040595347947L;
-       private final Multimap<PlayerID, String> alliances;
-
-       // Copy constructor is okay using private API
-       public SerializationProxy(AllianceTracker allianceTracker) {
-         alliances = ImmutableMultimap.copyOf(allianceTracker.alliances);
-       }
-
-       // This method MUST only use public APIs
-       protected Object readResolve() {
-         return new AllianceTracker(alliances);
-       }
-     }
- * </code>
+ * Code formatted below for easy copy paste:
```

Comment:
I hesitated to make this change because someone obviously thought about formatting it in such a way as to allow easy copy-paste.  I ended up making it non-copy-paste friendly **from the source editor** for the following reasons:

* It already contained a non-Java-source-compatible entity: namely, the `&#64;` character entity reference.
* The Eclipse Formatter produces the output as given in this PR.  If someone decided to apply the formatter across a set of files that included this one, they would have to revert this one change every time such an operation was performed.
* The "copy-paste friendly" version is available from the IDE's Javadoc view or hover (at least in Eclipse; presumably the same feature is available in IntelliJ).

Please advise if it should be reverted.  It will not re-introduce any Checkstyle violations, it will simply make the Eclipse formatter want to reformat it.